app/src/main/java/org/mozilla/rocket/chrome/di/ChromeModule.kt

```diff
@@ -86,6 +88,12 @@ object ChromeModule {
     @Provides
     fun provideBookmarkRepository(appContext: Context): BookmarkRepository = BookmarkRepository.getInstance(BookmarksDatabase.getInstance(appContext))
 
+    @JvmStatic
+    @Singleton
```

Comment:
if we use it as a singleton, do we still need the class as a singleton?

Reply:
fixed